planning/freespace_planning_algorithms/CMakeLists.txt

```diff
@@ -34,7 +34,13 @@ if(BUILD_TESTING)
   target_link_libraries(freespace_planning_algorithms-test
     freespace_planning_algorithms
   )
-
+  find_package(ament_cmake_ros REQUIRED)
+  ament_add_ros_isolated_gtest(freespace_planning_algorithms_no_obstacles-test
+    test/src/test_freespace_planning_algorithms_no_obstacles.cpp
+  )
+  target_link_libraries(freespace_planning_algorithms_no_obstacles-test
+    freespace_planning_algorithms
+  )
```

Comment:
do not add two "test" in CMake (`freespace_planning_algorithms-test` and `freespace_planning_algorithms_no_obstacles-test`).
Instead, please have one `freespace_planning_algorithms-test` with all test files.
You should also remove the `find_package(ament_cmake_ros REQUIRED)`.